pyproject.toml

```diff
@@ -9,8 +9,8 @@ description = "Lightspeed Stack providers for llama-stack"
 requires-python = ">=3.10"
 dependencies = ["llama-stack>=0.2.6", "httpx"]
 
-[tool.setuptools]
-packages = ["lightspeed_stack_providers"]
+[tool.setuptools.packages]
+find = {}
```

Comment:
If not doing this change, once building by using `pip -m build`, the subfolders are not included in the generated package.